src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -8,11 +8,11 @@
  */
  public class BattleshipModel {
 
-    private Civilian clipper = new Civilian("Clipper", 3, new Coordinate(0, 0), new Coordinate(0, 0));
-    private Civilian dinghy = new Civilian("Dinghy", 1, new Coordinate(0, 0), new Coordinate(0, 0));
-	private Military aircraftCarrier = new Military(false,"AircraftCarrier",5, new Coordinate(0,0),new Coordinate(0,0));
-    private Military battleship = new Military(true,"Battleship",4, new Coordinate(0,0),new Coordinate(0,0));
-    private Military submarine = new Military(true,"Submarine",2, new Coordinate(0,0),new Coordinate(0,0));
+    protected Civilian clipper = new Civilian("Clipper", 3, new Coordinate(0, 0), new Coordinate(0, 0));
+    protected Civilian dinghy = new Civilian("Dinghy", 1, new Coordinate(0, 0), new Coordinate(0, 0));
+	protected Military aircraftCarrier = new Military(false,"AircraftCarrier",5, new Coordinate(0,0),new Coordinate(0,0));
+    protected Military battleship = new Military(true,"Battleship",4, new Coordinate(0,0),new Coordinate(0,0));
+    protected Military submarine = new Military(true,"Submarine",2, new Coordinate(0,0),new Coordinate(0,0));
```

Comment:
Nice catch on changing everything from private to protected so the child class can access the variables.